docs/Resources-Appendices/ascii-collating-sequence.md

```diff
@@ -2,7 +2,8 @@
 sidebar_position: 1
 ---
 
-# ASCII Collating Sequence
+import Image from '@theme/IdealImage';
 
-![](/img/image5.png)
+# ASCII Collating Sequence
 
+<Image img={require('/img/image5.png')} />
```

Comment:
The `Image` component [supports `alt` text via a prop](https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-plugin-ideal-image/src/theme/IdealImage.js#L29), so let's include it here:

```js
<Image img={require('/img/image5.png')} alt='ASCII collating sequence table' />
```

In a follow-up, we should convert this image to text in a table instead, since it's totally unusable for users on screen readers.

Reply:
Thank you for your suggestion. File updated.

Reply:
@humphd  Is there already an existing issue for changing this image into a table? I can create one if no.